bindings/c/test/mako/mako.cpp

```diff
@@ -257,7 +297,6 @@ int runOneTransaction(Transaction& tx,
 				stats.incrErrorCount(op);
 			} else {
 				// abort
-				tx.reset();
```

Comment:
In `runOneTransaction`, I only removed the `tx.reset()` calls that looked like they were going to return to the caller. The other cases such as in lines 315 and 341 seemed appropriate to keep retrying/resetting on the same tenant because those resets seem like they are meant for error handling and retries rather than attempting to start a new set of operations